NppExec/src/DlgAdvancedOptions.cpp

```diff
@@ -158,7 +162,9 @@ INT_PTR CALLBACK AdvancedOptionsDlgProc(
         advOptDlg.OnInitDlg(hDlg);
     }
 
-    return 0;
+    // Note: This is greedy and must be the last handler
+    if (PickColorBtn::HandleMessageForDialog(hDlg, uMessage, wParam, lParam)) return true;
+    return false;

```

Comment:
To be precise, should be `return 0` since the return type is `INT_PTR`

Reply:
Yes and no :). The original DlgProc signature was actually `BOOL CALLBACK Dlg...` and was only changed when Windows was ported to 64-bit to deal with some messages that returned a pointer directly and there is no `BOOL_PTR `type. You can still see some places where it has not been updated, [here for example](https://learn.microsoft.com/en-us/windows/win32/dlgbox/dlgbox-programming-considerations#dialog-box-procedures).

The return value means, "yes" I handled the message, or "no", I did not, do the default. I can change them to int if you really want me to...